internal/oci/container.go

```diff
@@ -182,11 +191,11 @@ func (c *Container) FromDisk() error {
 
 	// this is to handle the situation in which we're upgrading
 	// versions of cri-o, and we didn't used to have this information in the state
-	if tmpState.InitPid == 0 && tmpState.InitStartTime == 0 && tmpState.Pid != 0 {
+	if tmpState.InitPid == 0 && tmpState.InitStartTime == "" && tmpState.Pid != 0 {
 		if err := tmpState.SetInitPid(tmpState.Pid); err != nil {
 			return err
 		}
-		logrus.Infof("PID information for container %s updated to %d %d", c.id, tmpState.InitPid, tmpState.InitStartTime)
+		logrus.Infof("PID information for container %s updated to %d %s", c.id, tmpState.InitPid, tmpState.InitStartTime)
```

Comment:
maybe pre-tea time, but won't tmpState.InitStartTime always be "" here?  Is that useful in the message?

Reply:
SetInitPid updates the two values for tmpState :)